### ISSUE TITLE
SecurityException: getDataNetworkTypeForSubscriber (Android 11)

### ISSUE THREAD

**Issue by DmytroKhmil** [mapbox/mapbox-navigation-android#3577]
**Android API:**
compileSdkVersion 30
targetSdkVersion 30
**Mapbox Navigation SDK version:**
1.0.0-rc.8

my problem is similar to already solved in [mapbox-gl-native-android](https://github.com/mapbox/mapbox-gl-native-android/issues/425)

```
java.lang.SecurityException: getDataNetworkTypeForSubscriber
        at android.os.Parcel.createExceptionOrNull(Parcel.java:2373)
        at android.os.Parcel.createException(Parcel.java:2357)
        at android.os.Parcel.readException(Parcel.java:2340)
        at android.os.Parcel.readException(Parcel.java:2282)
        at com.android.internal.telephony.ITelephony$Stub$Proxy.getNetworkTypeForSubscriber(ITelephony.java:8762)
        at android.telephony.TelephonyManager.getNetworkType(TelephonyManager.java:3021)
        at android.telephony.TelephonyManager.getNetworkType(TelephonyManager.java:2985)
        at com.mapbox.android.telemetry.TelemetryUtils.obtainCellularNetworkType(TelemetryUtils.java:165)
        at com.mapbox.mapboxsdk.module.telemetry.PhoneState.<init>(PhoneState.java:39)
        at com.mapbox.mapboxsdk.module.telemetry.TelemetryImpl.onAppUserTurnstileEvent(TelemetryImpl.java:46)
        at com.mapbox.mapboxsdk.maps.MapView.onCreate(MapView.java:303)
        at com.mapbox.navigation.ui.NavigationView.onCreate(NavigationView.java:127)
        at com.a.b.ui.navigationui.NavigationViewFragment.onViewCreated(NavigationViewFragment.kt:66)
        at androidx.fragment.app.FragmentStateManager.createView(FragmentStateManager.java:322)
        at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1185)
        at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1354)
        at androidx.fragment.app.FragmentManager.moveFragmentToExpectedState(FragmentManager.java:1432)
        at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1495)
        at androidx.fragment.app.BackStackRecord.executeOps(BackStackRecord.java:447)
        at androidx.fragment.app.FragmentManager.executeOps(FragmentManager.java:2167)
        at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:1990)
        at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:1945)
        at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:1847)
        at androidx.fragment.app.FragmentManager$4.run(FragmentManager.java:413)
        at android.os.Handler.handleCallback(Handler.java:938)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:223)
        at android.app.ActivityThread.main(ActivityThread.java:7656)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)

**Comment by Guardiola31337** [comment]
Hey @DmytroKhmil thanks for reaching out and report your issue.

This will be fixed by https://github.com/mapbox/mapbox-navigation-android/pull/3579 and available in the next release.

In the meantime, you can workaround OP adding `"com.mapbox.mapboxsdk:mapbox-android-telemetry:6.1.0"` as a dependency to your app's `build.gradle` file.

cc @tobrun @nagineni

**Comment by Guardiola31337** [comment]
Closing here as https://github.com/mapbox/mapbox-navigation-android/pull/3579 landed.

Thanks for checking out the SDK @DmytroKhmil 🙇‍♂️

**Comment by nanorus** [comment]
> Hey @DmytroKhmil thanks for reaching out and report your issue.
> 
> This will be fixed by #3579 and available in the next release.
> 
> In the meantime, you can workaround OP adding `"com.mapbox.mapboxsdk:mapbox-android-telemetry:6.1.0"` as a dependency to your app's `build.gradle` file.
> 
> cc @tobrun @nagineni

Thank you.
